.changeset/fuzzy-frogs-smash.md

```diff
@@ -0,0 +1,5 @@
+---
+"ledger-live-desktop": patch
+---
+
+My Ledger: app in app catalog, fix layout of items
```

Comment:
too much changesets?

Reply:
I prefer to make separate changesets for unrelated changes. It is not an issue with the changeset library:
https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md#you-can-add-more-than-one-changeset-to-a-pull-request